t/unit/app/test_beat.py

```diff
@@ -301,6 +301,24 @@ def test_info(self):
         scheduler = mScheduler(app=self.app)
         assert isinstance(scheduler.info, str)
 
+    def test_apply_entry_handles_empty_result(self):
+        s = mScheduler(app=self.app)
+        entry = s.Entry(name='a name', task='foo', app=self.app)
+        with (patch.object(s, 'apply_async') as mock_apply_async,
+              patch("celery.beat.debug") as mock_debug):
+            mock_apply_async.return_value = None
+            s.apply_entry(entry)
+        mock_debug.assert_called_once_with('%s sent.', entry.task)
+
+        task_id = 'taskId123456'
+        with (patch.object(s, 'apply_async') as mock_apply_async,
+              patch("celery.beat.debug") as mock_debug):
```

Comment:
oh woops - this syntax (parenthesized context managers) was not available in older python versions. fixing now...